### DISCUSSION TITLE
Issues when HA has two network adapters

### DISCUSSION THREAD

**Discussion by iddora** [fuatakgun/eufy_security#231] [Q&A]
I'm running HA on a RPi4 that is configured with two network adapters.
I suspect this is causing the eufy_security_addon to fail finding my homebase 2 unit.
I was looking for a way to configure which network adapter the addon should use in order to connect to the network that my eufy products are on, but couldn't find any.

![image](https://user-images.githubusercontent.com/10020983/150656220-1d374e1a-4377-415e-bafe-8bd1c5e2a4b7.png)

I'm assuming that this is the cause of my issues, as I have run to similar issues in the past with the HomeKit integration.
I solved it by configuring the "ip_address" parameter [https://www.home-assistant.io/integrations/homekit/](url)

Help would be much appreciated.

**Comment by iddora** [comment]
Seems like my assumption was wrong.
I have connected my HomeBase2 to the second network and the security add-on still does not find it.

How can I debug this?

**Comment by fuatakgun** [comment]
Hey i don't have any experience nor knowledge on this topic, this looks like beyond the integration itself.
